ignite/services/chain/serve.go

```diff
@@ -222,9 +222,17 @@ func (c *Chain) setup() error {
 // dependencies and pre-conditions.
 func (c *Chain) checkSystem() error {
 	// check if Go has installed.
-	if !xexec.IsCommandAvailable("go") {
+	if !gocmd.Available() {
 		return errors.New("Please, check that Go language is installed correctly in $PATH. See https://golang.org/doc/install")
 	}
+	// Check version>=1.17
+	ok, err := gocmd.IsMinVersion("1.17")
```

Comment:
The `go install` command inside a module only works in version>=1.17

Reply:
That's totally fine, we are enforcing new chains to use go 1.18 in the `go.mod` anyways

Reply:
Oh damn, I didn't think of that. Maybe all those adds about checking go version should be removed then...

Reply:
yeah, I think we should remove

Reply:
OK ty @aljo242 !
 done here [f8d2e30](https://github.com/ignite/cli/pull/2803/commits/f8d2e30c9d500b4fc7f8722f19c0dee5af8a7bfc)